### DISCUSSION TITLE
MAUI iOs app is throwing exception System.DllNotFoundException: @rpath/libSkiaSharp.framework/libSkiaSharp

### DISCUSSION THREAD

**Discussion by scrmorph** [Live-Charts/LiveCharts2#946] [General]
I cannot deploy chart on iOs as MAUI app is always throwing that exception. 
I tried to add SkiaSharp separately but it didn't help. Anyone knows how to ix this issue? 

MAUI Windows App works ok, no issues.

**Comment by stevebrooke3000** [comment]
Not getting an exception, but ...
Do NOT see charts on local iPhone 7. Other controls (buttons seem OK). 
I do see the charts with the iOS simulator, physical Android and Windows. The iPhone 7 has iOS 15.7.3, the iOS simulators were iPhone 7 iOS 15.5 and iPhone 8  iOS 16.2
Charts were visible initially with the local iPhone 7, but then stopped working after a build was tried. I reverted the changes and rebuilt but no charts. I thought that maybe LiveCharts had updated, so I uninstalled and tried older versions, same result. The current version I am using is 2.0.0-beta.700
The app seems to be working with no exceptions or crashes, but charts are not visible on the local device
Any ideas?

**Reply by stevebrooke3000** [reply]
Also meant to add that I tried with the iPhone 7 reset to factory.

**Reply by JerryAndersen** [reply]
I experience the same with my iPhone13Pro and iOS 16.5
